src/rich_cli/__main__.py

```diff
@@ -102,7 +102,7 @@ def read_resource(path: str, lexer: Optional[str]) -> Tuple[str, Optional[str]]:
         if path == "-":
             return (sys.stdin.read(), None)
 
-        with open(path, "rt") as resource_file:
+        with open(path, "rt", encoding="utf8") as resource_file:
```

Comment:
It fixes the described issue, but... Is doing this really a good idea? I'm not sure, to be honest 😅

Reply:
The annoying truth is that you can't be sure what the encoding will be. Can you also add `errors="replace"` in the off change there are utf-8 encoding errors.

At some point we might use `chardet` to give us extra confidence.

Reply:
TIL: :mortar_board: the Python `open` function can indeed accept a `errors` arg that specifies how encoding and decoding errors are to be handled - which is really handy! :slightly_smiling_face: 
Done in [53cb0e9](https://github.com/Textualize/rich-cli/pull/56/commits/53cb0e9f8ddcfccafc2073a05370928fed2bb8b9)